### DISCUSSION TITLE
Example 1/2: UE location not working

### DISCUSSION THREAD

**Discussion by david-opt** [InterDigitalInc/AdvantEDGE#432] [Q&A] [answered]
Hello,

I have installed the latest version 1.9.1 of AdvantEdge. All services are executed without errors. In the example, the external UE always invokes the same service instance (the image also remains permanently the same). I have tried all network locations by the mobility event, but the invoked service instance always remains the same. In version 1.8 this service instance change still happened. Did you remove the feature or am I doing something wrong?

Kind regards

**Comment by dilallkx** [comment, marked as answer]
Hi @david-opt,

Thank you for reaching out!

I was able to reproduce the issue on my end, and this is in fact a bug with the latest release. The problem is caused by a change in the iptables (netfilter) version used with the Debian baseline container image upgrade in AdvantEDGE v.1.9.1. The iptables go package should also have been upgraded in v1.9.1, but this was not done, which lead to an error in the traffic routing rules causing the behavior you observed.

I have updated the iptables package in the new [v1.9.2](https://github.com/InterDigitalInc/AdvantEDGE/releases/tag/v1.9.2) release. Please give it a try and let me know if this fixes your issue.

Best regards!

**Reply by david-opt** [reply]
Hey @dilallkx,

great, everything is working again. Thank you for fixing the problem so quickly.

Best regards!
